cpu/stm32f3xx/cdc_acm/usb_pwr.c

```diff
@@ -37,7 +37,7 @@
 /* Private macro -------------------------------------------------------------*/
 /* Private variables ---------------------------------------------------------*/
 __IO uint32_t bDeviceState = UNCONNECTED; /* USB device status */
-__IO bool fSuspendEnabled = FALSE;  /* true when suspend is possible */
+__IO bool fSuspendEnabled = TRUE;  /* true when suspend is possible */
```

Comment:
Ensuring that the drivers used on the app and bootloader are configured in exactly the same manner.  

For 1.0.0, I'd like to do a review of the USB stack in it's entirety.